examples/methods/plot_subspace.py

```diff
@@ -0,0 +1,347 @@
+        ax.set_title("Accuracy on source", fontsize=12)
+        ax.text(
+            x_max - 0.3,
+            y_min + 0.3,
+            ("%.2f" % score).lstrip("0"),
+            size=15,
+            horizontalalignment="right",
+        )
+
+    figure.suptitle(suptitle, fontsize=16, y=1)
+    figure.tight_layout()
+
+
+clf = base_classifier
+clf.fit(Xs, ys)
+create_plots(
+    base_classifier, "Without DA", suptitle="Illustration of the classifier with no DA"
+)
+
+
+# %%
+#     Illustration of the Subspace Alignment method
+# ------------------------------------------
+#
+# As we assume that the  source and target domains are represented
+# by subspaces described by eigenvectors;
+# This method seeks a domain adaptation solution by learning a mapping
+# function which aligns the source subspace with the target one.
+#
+# See [1] for details:
+#
+# .. [1] Basura Fernando et. al. Unsupervised Visual
+#        Domain Adaptation Using Subspace Alignment.
+#        In IEEE International Conference on Computer Vision, 2013.
+
+clf = SubspaceAlignment(base_classifier, n_components=1)
+clf.fit(X, y, sample_domain=sample_domain)
+create_plots(clf, "Subspace Alignment")
+
+# %%
+#     Illustration of the Transfer Component Analysis method
+# ------------------------------------------
+#
+# The goal of Transfer Component Analysis (TCA) is to learn some transfer
+# components across domains in a reproducing kernel Hilbert space using Maximum
+# Mean Discrepancy (MMD)
+#
+# See [2] for details:
+#
+# .. [2] Sinno Jialin Pan et. al. Domain Adaptation via
+#        Transfer Component Analysis. In IEEE Transactions
+#        on Neural Networks, 2011.
+
+clf = TransferComponentAnalysis(base_classifier, n_components=1, mu=2)
+clf.fit(X, y, sample_domain=sample_domain)
+create_plots(clf, "TCA")
+
+# %%
+#     Illustration of the Transfer Joint Matching method
+# ------------------------------------------
+#
+# In most of the previous works, we explored two learning strategies independently for
+# domain adaptation: feature matching and instance reweighting. Transfer Joint Matching
+# (TJM) aims to use both, by adding a constant to tradeoff between to two.
+#
+# See [3] for details:
+#
+# .. [3] Long et al., 2014] Long, M., Wang, J., Ding, G., Sun, J., and Yu, P. (2014).
+#         Transfer joint matching for unsupervised domain adaptation. In IEEE Conference
+#         on Computer Vision and Pattern Recognition (CVPR), pages 1410–1417.
+
+clf = TransferJointMatching(
+    base_classifier, kernel="linear", tradeoff=0, n_components=1, max_iter=20, tol=0.1
+)
+clf.fit(X, y, sample_domain=sample_domain)
+create_plots(
+    clf,
+    "TransferJointMatching with linear kernel",
+)
+
+clf = TransferJointMatching(
+    base_classifier, kernel="rbf", tradeoff=0.1, n_components=1, max_iter=20, tol=0.1
+)
+clf.fit(X, y, sample_domain=sample_domain)
+create_plots(
+    clf,
+    "TransferJointMatching with rbf kernel",
+)
```

Comment:
You should put the same kernel as in the other methods, i.e. `rbf`, and remove `linear`.

Reply:
Also, let the default parameters (except `n_components`).

Reply:
the default parametter means no tradeoff, it would be a shame as the whole point of thie method is to be able to have this tradeoff, but for the rest, I'll do as you said